main.go

```diff
@@ -47,7 +50,7 @@ func main() {
 	}
 }
 
-func parseTemplates(templatePaths []string, leftLimiter string, rightLimiter string, continueOnMissingKey bool, targetBasePath string, templateBasePath string, secrets map[string]string) error {
+func parseTemplates(templatePaths []string, leftLimiter string, rightLimiter string, continueOnMissingKey bool, targetBasePath string, templateBasePath string, secrets map[string]map[string]string) error {
```

Comment:
I was a bit confused by this function name, because it does not only parse templates, but it actually executes them as well and writes the results to files, so I think this function should get renamed at some point.